src/Reflection/ReflectionFunctionAbstract.php

```diff
@@ -188,6 +190,21 @@ public function isVariadic(): bool
         return false;
     }
 
+    /**
+     * Check if the function can throw error/exception.
+     */
+    public function isCanThrows(): bool
```

Comment:
Not sure why the `is` wording is used :thinking: 

`couldThrow()` IMO

Reply:
@Ocramius ok, updated